decidim-censuses/app/mailers/decidim/censuses/application_mailer.rb

```diff
@@ -0,0 +1,10 @@
+module Decidim
```

Comment:
If we are not going to send emails, I would remove this class (of everything keeps working fine, of course 😅)